### DISCUSSION TITLE
disable gtk3 create document suggestions

### DISCUSSION THREAD

**Discussion by weter11** [lah7/gtk3-classic#64] [Help & Support] [answered]
Can't find a way to disable old used names for files and folders in thunar file manager.

![Снимок экрана_2022-02-02_10-18-41](https://user-images.githubusercontent.com/14630689/152127093-8b2d9296-c84a-4f5f-9e88-ae8e5ba618f2.png)

This is `File - Create document` dialogue in thunar. Or this is thunar problem, I tried search where it stores that file names, but `grep -rnw `don't find anything.

**Comment by lah7** [comment, marked as answer]
The "Create Document" menu is normally populated by the **Templates** folder. Check your `~/.config/user-dirs.dirs` file as the path might be pointing elsewhere. The default is:

    XDG_TEMPLATES_DIR="$HOME/Templates"

**Comment by weter11** [comment]
Looks like I don't properly formulate my question for searching in google. I searched for bugs in gtk, but it's not a bug.
Thanks.
